dds/idl/marshal_generator.cpp

```diff
@@ -2378,12 +2393,57 @@ bool marshal_generator::gen_struct(AST_Structure* node,
       }
     }
 
+    string exp, fn = " { return false; }";
+    if (octetSeqOnly) {
+      const ACE_CString exporter = be_global->export_macro();
+      if (exporter != "") {
+        exp = string(" ") + exporter.c_str();
+      }
+      fn = ";";
+      string field_name = fields[0]->local_name()->get_string();
+
+      const char* get_len, *set_len, *get_buffer, *buffer_pre = "";
```

Comment:
Please separate these into different lines as it says in the guidelines.

Reply:
I think that was my commit... sorry.